backend/cfg/cfgize.ml

```diff
@@ -66,7 +70,8 @@ end = struct
       catch_handlers;
       next_instruction_id;
       iends_with_poptrap;
-      exception_handlers
+      exception_handlers;
+      cold_blocks = Label.Tbl.map blocks (fun _ -> false)
```

Comment:
(I am not positive it would be better
with only one table, but is there a
benefit to two tables? Could we even
have `cold` be a field of `basic_block`?)

Reply:
I can do that; yes. In this case, I can even avoid using the section mechanism and simply use `reorder_blocks`; it might even be simpler. What do you think is best?